charters/DataProcessingService/DataProcessingPipelines.md

```diff
@@ -0,0 +1,70 @@
+# [HCA Data Processing Pipelines](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+The HCA Data Processing Pipelines are cloud-based and portable analysis pipelines that transform the data in the Human Cell Atlas.
+
+## Objectives
+The data processing service aims to develop scientific analysis pipelines for all analyses required on all scientific data types in the Human Cell Atlas Data Coordination Platform (DCP). Pipelines for additional data types, analyses, and improvements to these pipelines will be added iteratively and in collaboration with the HCA Analysis Working Group (AWG) and the community. 
+
+## Definitions
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports
+__Onboarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay 
+__Standardized Pipeline:__ An official pipeline of the HCA project 
+
+## In-scope
+* Scoping or review of scoping for new pipelines  
+* Creating all standardized pipelines for scientific data 
+* Reviewing, on-boarding, and potentially creating community pipelines  
```

Comment:
I agree it would be great to document answers to those questions. 

>When would we create community pipelines, when would we not? 

I think this question is answered in the definition of a `community pipeline` above.

> What do we hope a community pipeline looks like? 

I'm not sure what this means, outside of the statement in the definition above that it is a pipeline that can run in the DCP but is not a `Standard Pipeline`. Could you elaborate?

> Who are we committing to work with? 

Would this be answered by updating the line to:
`* Reviewing, on-boarding, and potentially creating community pipelines for assays prioritized by the HCA DCP Governance Group`

> Will there be efforts to generalize this process to make it easier for external users to bring pipelines to the DCP?

I do think we should do what we can to help with this, especially surrounding documentation and workshops, which I agree should be highlighted explicitly somewhere. @jodihir Do you think that belongs here as well as in the DCP Communications charter, or only here? I'll add that. 

I do not think the mint team will have the bandwidth to do this topic its full justice, however. I do think the Emerald Empire and Cromwell Interactions team could help and do have this on their minds for future prioritization. I don't think it's an immediate priority and I don't know what those efforts would look like, so I don't know that I'd add more detail here than what I mentioned above.

Thoughts? My other idea is that there could be a separate RFC or Process Charter on community pipelines if we want to expand on the idea further in a public forum.

Reply:
I am glad you are excited about this, but it is still not fully understood. I feel we will grow this in the coming year. I am not sure this level of detail is needed in the charter, I would, however, be excited about making other documentation and guidelines that move us towards exercising this concept. If we learn from the community pipeline effort things that would require the charter to be updated, that is a possibility.